pkg/metautil/peer.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metautil
+
+import (
+	"github.com/pingcap/kvproto/pkg/metapb"
+)
+
+// IsLearner judges whether the Peer's Role is Learner.
+func IsLearner(peer *metapb.Peer) bool {
```

Comment:
Why do we need a `metautil` package, can we put it into `core/region.go`?

Reply:
I'll put it into `core/meta.go`.